src/providers/app-user.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
+import { Http } from '@angular/http';
+import 'rxjs/add/operator/map';
+
+/*
+  Generated class for the AppUser provider.
+
+  See https://angular.io/docs/ts/latest/guide/dependency-injection.html
+  for more info on providers and Angular DI.
+*/
+@Injectable()
+export class AppUser {
+
+  constructor(public http: Http) {
+    console.log('Hello AppUser Provider');
+  }
+  
+  // CHANGE THE BASE URL TO MATCH YOUR STRONGLOOP API EXPLORER
+  baseUrl: string = "http://sp-17-vincent-jbrownssf.c9users.io:8080/api"
```

Comment:
This is where each person needs to change the baseUrl to point to their own back end.